cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/encryption/EncryptedSnapshotService.java

```diff
@@ -51,6 +50,8 @@
 
```

Comment:
Actually, if this class indeed was to be dropped, I wonder if the enum constants `AWS_SNAPSHOT` and `AWS_ENCRYPTED_VOLUME` are still needed. If not used anywhere after class cleanup, they can be also removed.
@doktoric @biharitomi Any concerns?

Reply:
Checked for the usage of `EncryptedSnapshotService`, It can be dropped as not used anywhere else(apart from already removed code.) Along with, `EncryptedSnapshotServiceTest.java` and `deleteResources()` called by `AwsTerminateService.cleanupEncryptedResources()`. Thank you @lajosrodek